app/models/booking.rb

```diff
@@ -1,4 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :task
+  validates  :user, :task, presence: true
```

Comment:
do user and task go on separate lines? not sure, looking for validation from the group here for my learning.

Reply:
nope you can put all of them on single line:)